### PR TITLE
[stable/prometheus] add livenessProbe and readinessProbe

### LINKED ISSUE #3000: [prometheus] use other endpoints for readiness/liveness
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

**Version of Helm and Kubernetes**: v2.7.0 / v1.7.8-gke.0

**Which chart**: Prometheus

**What happened**: the readinessProbe is set on the `/status` endpoint

**What you expected to happen**: the readinessProbe to be set on `/-/ready` endpoint

**How to reproduce it** (as minimally and precisely as possible): -

**Anything else we need to know**: There is also another endpoint: `/-/healthy` that might be useful.

See [prometheus/prometheus#2831](https://github.com/prometheus/prometheus/issues/2831) for details.

### PR DESCRIPTION
**Pull request by thuandt**
Use /-/healthy for livenessProbe and /-/ready for readinessProbe
Fix #3000

### CODE REVIEW

File: stable/prometheus/Chart.yaml
Comment (foxish) [inline stable/prometheus/Chart.yaml]: Why the 2 version bump?

**Review by foxish — COMMENTED** [review]
It also pins it to a newer version of prometheus correct? Given comment https://github.com/prometheus/prometheus/issues/2831#issuecomment-334536817.
Should that be a part of the readme?

File: stable/prometheus/Chart.yaml
Comment (thuandt) [reply stable/prometheus/Chart.yaml]: ah. I bump to `4.6.14` in another PR. If this PR merged first, it should be `4.6.14` instead. 

Current Prometheus version in chart is `v1.8.1` which already support 2 endpoint, i think we don't need put it in readme anyway

**Comment by thuandt** [comment]
@foxish I rebased from master branch. Please help me review! Thanks

**Comment by foxish** [comment]
@mgoodness, you're the maintainer of the chart, PTAL. LGTM, will merge once @mgoodness takes a look.

**Comment by foxish** [comment]
Sorry @thuandt, this fell off the review queue - and looks like it has a merge conflict now. PTAL

**Comment by thuandt** [comment]
@foxish Chart version has been increased. PR should be ok now.

**Review by matthope — APPROVED** [review]

**Pull request merged**
